README.md

```diff
@@ -113,6 +113,13 @@ To change the endpoints you can modifiy the AppSettings.cs file in the SmartHote
 ## Demo Scripts
 There is a detailed step-by-step demo script in the [demoscripts](https://github.com/Microsoft/SmartHotel360-mobile-desktop-apps/tree/master/DemoScripts) folder of this repo that you can use in your own presentations.
 
+## IoT Demo - Comming Soon
+
+1. In order to have live data working and enable room controls you must setup the [IoT Demo Backend](https://github.com/Microsoft/SmartHotel360-AKS-DevSpaces-Demo/blob/helm/README.md)
+2. Get Room Devices Api endpoint (setup via step 1) and change the value of DefaultRoomDevicesEndpoint in AppSettings.cs
+3. Get the Id of Room 11 (setup via step #1) and change the value of DefaultRoomId in AppSettings.cs
+4. Set property DefaultUseFakes = false in AppSettings.cs
```

Comment:
Do we really need DefaultUseFakes set to false in order to function?

Reply:
I think so because it's used at different places in the app.

Reply:
ok

Reply:
Looks like the latest changes from MS put this back to defaulting to false. Maybe we can say **4. Ensure that the property DefaultUseFakes = false in AppSettings.cs**

This way we cover if they keep flipping it back and forth